pages/ksphere/kommander/latest/licensing/index.md

```diff
@@ -1,35 +1,28 @@
 ---
 layout: layout.pug
 navigationTitle: Licensing
-title: Licensing
+title: License 
 excerpt:
 menuWeight: 6
 ---
 
-## Licensing
+## Purchase a License
 
-Kommander requires a valid license to continue use beyond adding a second cluster.
+Kommander requires a valid license if you want to add more than two clusters. 
```

Comment:
Should we call out that Kommander cluster itself counts as a cluster?

> You can evaluate Kommander running two clusters for free. Kommander cluster plus one worker cluster. If you want to add more clusters you need a valid license.

Reply:
+1

Reply:
"worker cluster" is not a term I've heard before and I don't want to overload worker node pools. Went with "additional cluster"